searx/search/processors/abstract.py

```diff
@@ -91,7 +97,7 @@ def _extend_container_basic(self, result_container, start_time, search_results):
     def extend_container(self, result_container, start_time, search_results):
         if getattr(threading.current_thread(), '_timeout', False):
             # the main thread is not waiting anymore
-            self.handle_exception(result_container, 'Timeout', None)
+            self.handle_exception(result_container, 'timeout', None)
```

Comment:
`'timeout'` matches the key `timeout` of `exception_classname_to_text` (webapp.py)